slides/lesson2/solution/versions.tf

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
styling: let's remove the extra line

Reply:
we can do this in the next cycle this is not a blocker, thanks for pointing that out

Reply:
This will be done in another PR. Created an issue to keep track of this #33